File: 1º Semestre/Progr Comp 1/aula7/ex3.c
// Desenvolva um programa em C que solicite ao usuário informar 20 números inteiros. 
// O programa em C deverá mostrar na tela o menor número digitado e a quantidade de números que são iguais a este número.

#include <stdio.h>

int main() {
   int x,num,menor,cont;

   for (x=1;x<=20;x++) {
      printf("Digite o numero %d: ",x);
      scanf("%d",&num);

      if (x==1) {
         menor=num;
         cont=1;
      }

      if (num<menor) {
         menor=num;
         cont=1;
      } else if (num == menor) {
         cont++;
      }
   }

   printf("Menor: %d\nQauntidade Menor: %d",menor,cont);

   return 0;
}

File: 1Semestre/Introd Comp e Tec/aula2/ex3.py
# Desenvolva um programa em Python que solicite ao usuário digitar um número inteiro. 
# Após a digitação, o computador deverá mostrar na tela uma mensagem informando se o número maior, menor ou igual a zero.
# Exemplos: se o usuário informar o número 38, deverá ser mostrado na tela a mensagem “38 é um número maior que zero”. 
# Se o usuário informar o número -13, deverá ser mostrado na tela a mensagem “-13 é um número menor que zero”. 
# Se o usuário informar o número 0, deverá ser mostrado na tela a mensagem “Número digitado igual a zero”. 

numero = int(input("Digite um número: "))

if numero > 0:
    print(numero,"é um número maior que zero")
elif numero == 0:
    print("Número digitado igual a zero")
else:
    print(numero,"é um número menor que zero")